actiontext/lib/templates/installer.rb

```diff
@@ -8,6 +10,9 @@
 JS_DEPENDENCIES = JS_PACKAGE["peerDependencies"].dup.merge \
   JS_PACKAGE["name"] => "^#{JS_PACKAGE["version"]}"
 
+say "Installing JavaScript dependencies"
+run "bin/yarn add #{JS_DEPENDENCIES.map { |name, version| "#{name}@#{version}" }.join(" ")}"
```

Comment:
Please add `abort_on_failure: true` option to `run` action.

Reply:
By the way, what happens if people try to use Action Text outside of Rails? Then they probably won’t have bin/yarn. So perhaps Action Text should add that too.

Reply:
Do you think it would make sense to check if bin/yarn exists, create it via template if it doesn't and then finally use it for running yarn?

Reply:
I guess that this installer is only supposed to be executed inside a Rails application (directory structure and dependent tasks are based on Rails application). 
For this reason, I do not think it is supposed to be executed outside the Rails application. If support it, we will need to review the whole task.